hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/components/HawkbitUIErrorHandler.java

```diff
@@ -38,10 +42,21 @@ public void error(final ErrorEvent event) {
 
         final Optional<Page> originError = getPageOriginError(event);
 
+        final HawkbitErrorNotificationMessage message = buildNotification(getRootExceptionFrom(event));
+
         if (originError.isPresent()) {
-            final HawkbitErrorNotificationMessage message = buildNotification(getRootExceptionFrom(event));
+            final Connector connector = ((ConnectorErrorEvent) event).getConnector();
```

Comment:
It has to be, because otherwise the originError would be null. Have a look in the method getPageOriginError and inside that in findAbstractComponent...